src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Controller/ServiceController.php

```diff
@@ -95,8 +111,16 @@ public function overviewAction()
             return $this->redirectToRoute('service_add');
         }
 
+
+        $serviceObjects = [];
+        foreach ($services as $service) {
+            $entityList = $this->entityService->getEntityListForService($service);
```

Comment:
Please read [this](https://www.pivotaltracker.com/story/show/162298797/comments/197068977)